aws/terraform/modules/metaflow/modules/metadata-service/api-gateway.tf

```diff
@@ -0,0 +1,181 @@
+  http_method = aws_api_gateway_method.this.http_method
+  status_code = 200
+  depends_on  = [aws_api_gateway_integration.this]
+}
+
+resource "aws_api_gateway_integration" "this" {
+  rest_api_id = aws_api_gateway_rest_api.this.id
+  resource_id = aws_api_gateway_resource.this.id
+  http_method = aws_api_gateway_method.this.http_method
+
+  request_parameters = {
+    "integration.request.path.proxy" = "method.request.path.proxy"
+  }
+
+  type                    = "HTTP_PROXY"
+  uri                     = "http://${aws_lb.this.dns_name}/{proxy}"
+  integration_http_method = "ANY"
+  passthrough_behavior    = "WHEN_NO_MATCH"
+  connection_type         = "VPC_LINK"
+  connection_id           = aws_api_gateway_vpc_link.this.id
+}
+
+resource "aws_api_gateway_integration" "db" {
+  rest_api_id = aws_api_gateway_rest_api.this.id
+  resource_id = aws_api_gateway_resource.db.id
+  http_method = aws_api_gateway_method.db.http_method
+
+
+  type                    = "HTTP_PROXY"
+  uri                     = "http://${aws_lb.this.dns_name}:8082/db_schema_status"
+  integration_http_method = "GET"
+  passthrough_behavior    = "WHEN_NO_MATCH"
+  connection_type         = "VPC_LINK"
+  connection_id           = aws_api_gateway_vpc_link.this.id
+}
+
+resource "aws_api_gateway_method_response" "this" {
+  rest_api_id = aws_api_gateway_rest_api.this.id
+  resource_id = aws_api_gateway_resource.this.id
+  http_method = aws_api_gateway_method.this.http_method
+  status_code = "200"
+  depends_on  = [aws_api_gateway_integration.this]
+}
+
+resource "aws_api_gateway_method_response" "db" {
+  rest_api_id = aws_api_gateway_rest_api.this.id
+  resource_id = aws_api_gateway_resource.db.id
+  http_method = aws_api_gateway_method.db.http_method
+  status_code = "200"
+  depends_on  = [aws_api_gateway_integration.db]
+}
+
+resource "aws_api_gateway_deployment" "this" {
+  rest_api_id = aws_api_gateway_rest_api.this.id
+
+  # explicit depends_on required to ensure module stands up on first `apply`
+  # otherwise a second followup `apply` would be required
+  # can read more here: https://stackoverflow.com/a/42783769
+  depends_on = [aws_api_gateway_method.this, aws_api_gateway_integration.this]
+
+  # ensures properly ordered re-deployments occur
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+resource "aws_api_gateway_stage" "this" {
+  deployment_id = aws_api_gateway_deployment.this.id
+  rest_api_id   = aws_api_gateway_rest_api.this.id
+  stage_name    = local.api_gateway_stage_name
+
+  tags = var.standard_tags
+}
+
+resource "aws_api_gateway_api_key" "this" {
+  count = var.api_basic_auth ? 1 : 0
+  name  = local.api_gateway_key_name
+
+  tags = var.standard_tags
+}
+
+resource "aws_api_gateway_usage_plan" "this" {
+  count = var.api_basic_auth ? 1 : 0
+  name  = local.api_gateway_usage_plan_name
+
+  api_stages {
+    api_id = aws_api_gateway_rest_api.this.id
+    stage  = aws_api_gateway_deployment.this.stage_name
```

Comment:
Did you mean `stage = aws_api_gateway_stage.this.stage_name` here?

Reply:
yes that makes more sense 👍🏻  I'll update